deformdemo/test.py

```diff
@@ -262,12 +262,38 @@ def setUpModule():
 
         browser = Chrome()
         return browser
+
     elif driver_name == "phantomjs":
         # TODO: Test fails on Phantomjs
         # They just hang in some point
         from selenium.webdriver import PhantomJS
 
         browser = PhantomJS()
+
+    elif (
+        driver_name == "selenium_standalone_firefox"
+        and os.environ.get('TRAVIS') != 'true'
```

Comment:
That is for future use and clarity.

TRAVIS environment variable automatically is set by travis-ci and shall not be set by user.
This is the way we identify if test is being run in local system or in travis-ci.

If TRAVIS is set to true, it means test is being run in travis environment.

Some time in future I am going to change [else clause](https://github.com/Pylons/deformdemo/blob/master/deformdemo/test.py#L271) to os.environ.get('TRAVIS') == 'true'.

At the moment we are only providing option to run selenium container in local system for a faster start up in development.

We could run selenium container in travis as well, but I am not seeing any benefit to make change in travis, since the code running in travis is hidden to user. they don't see if it is using container or the driver.

Reply:
OK, thanks for the explanation.